### ISSUE TITLE
How to draw the feature maps in Figure 2

### ISSUE THREAD

**Issue by wonlp** [c-yn/FocalNet#5]
hello, I want to draw a middle feature map same as the Figure 2. Could you tell me how to draw it? Thank you!

**Comment by c-yn** [comment]
Hi, given a feature map _visual_, the snippet is as follows:
 
`  plt.figure(dpi=300)`
`  visual = torch.mean(visual, dim=1)`
`  plt.imshow(visual.squeeze(0).detach().cpu().numpy())`
`  plt.axis('off')`
`  plt.show()`

**Comment by wonlp** [comment]
ok, I got it. Thank you.
